Assignment1/src/app/components/credit-card/credit-card-details/credit-card-details.component.html

```diff
@@ -1 +1,65 @@
-<p>credit-card-details works!</p>
+<div class="wrapper">
+  <div class="container-1" *ngIf="cardDetails$">
+    <table>
+      <thead>
+        <tr>
+          <th scope="col">Card Holder Name</th>
+          <th scope="col">Card Number</th>
+          <th scope="col">Csc Code</th>
+          <th scope="col">Expiration Month</th>
+          <th scope="col">Expiration Year</th>
+          <th scope="col">Issuer</th>
+          <th scope="col">Delete Card</th>
+          <th scope="col">Transactions</th>
+        </tr>
+      </thead>
+      <tbody>
+        <td data-label="Card Holder Name">
+          {{ cardDetails$.cardholder_name }}
+        </td>
+        <td data-label="Card Number">{{ cardDetails$.card_number }}</td>
+        <td data-label="Csc Code">{{ cardDetails$.csc_code }}</td>
+        <td data-label="Expiration Month">
+          {{ cardDetails$.expiration_date_month }}
+        </td>
+        <td data-label="Expiration Year">
+          {{ cardDetails$.expiration_date_year }}
+        </td>
+        <td data-label="Issuer">{{ cardDetails$.issuer }}</td>
+        <td data-label="Delete Card">
+          <button (click)="deleteCreditCard()">❌</button>
```

Comment:
Ved godt at du ikke fik material til at virke, men her vil det fx være oplagt at bruge et icon kryds istedet for en emoji :-)